f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?><record_update>
     <sys_dictionary action="INSERT_OR_UPDATE" element="current_contract_and_recurring_information" table="x_g_dis_atat_acquisition_package">
-        <active>false</active>
+        <active>true</active>
```

Comment:
@DGabriel-CCPO What happened here?  How did the Current Contract Information table change?

Reply:
Looks like the entire table was made active, but wasn't it already active?

Reply:
@jasonburkert-ccpo For some reason when I checked the able on my branch it wasn't active but i didnt disable and figured it must have been set to false in on of the recent changes

Reply:
OK thanks!

Reply:
After some testing re: data migration this was set to `true` in #383